### PR TITLE
add  SoftSign Activation

### PR DESCRIPTION
**Pull request by yajiedesign**
I am not find way to test Activation,Where should I add?

### CODE REVIEW

**Comment by piiswrong** [comment]
test_operators.py

File: src/operator/activation-inl.h [hunk 1]
```diff
@@ -82,7 +88,27 @@ class ActivationOp : public Operator {
...
+      break;
+    case activation::kTanh:
+      Assign(m_in_grad, req[activation::kData],
+             F<mshadow_op::tanh_grad>(m_out_data) * m_out_grad);
+      break;
+    case activation::kSoftReLU:
+      Assign(m_in_grad, req[activation::kData],
+             F<mshadow_op::softrelu_grad>(m_out_data) * m_out_grad);
+      break;
+    case activation::kSoftSign:
```
Comment (vchuravy) [inline src/operator/activation-inl.h]: Maybe we can reduce the code duplication, by just special casing `SoftSign`? Also by using `<type_traits>` and `is_same` we maybe don't need to introduce `param_`.

**Comment by tqchen** [comment]
let use consider move future unary activation function to https://github.com/dmlc/mxnet/blob/master/src/ndarray/unary_function-inl.h instead, they will be exposed to both ndarray and operators, and needs shorter code

**Comment by chuckbasstan123** [comment]
Is this "softsign" operation already merged to mxnet github master? I didn't see this operation anywhere. @yajiedesign

**Pull request closed without merging**